recipes/txtorcon/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "txtorcon" %}
+{% set version = "0.19.3" %}
+{% set sha256 = "f73396667909a3c7a98f4dd865edf4ed6a2518ee5a935d92e18b8a479ec244fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - twisted >=15.5.0
+    - ipaddress >=1.0.16  # [py2k]
+    - zope.interface >=3.6.1
+    - incremental
+    - automat
+
+test:
+  requires:
+    - pyopenssl
```

Comment:
I got the following error without it:
```python
import: 'txtorcon'
Traceback (most recent call last):
  File "/home/christopher/mc/conda-bld/txtorcon_1498584467192/test_tmp/run_test.py", line 2, in <module>
    import txtorcon
  File "/home/christopher/mc/conda-bld/txtorcon_1498584467192/_t_env/lib/python3.5/site-packages/txtorcon/__init__.py", line 16, in <module>
    from txtorcon.controller import connect
  File "/home/christopher/mc/conda-bld/txtorcon_1498584467192/_t_env/lib/python3.5/site-packages/txtorcon/controller.py", line 35, in <module>
    from txtorcon.endpoints import TorClientEndpoint, _create_socks_endpoint
  File "/home/christopher/mc/conda-bld/txtorcon_1498584467192/_t_env/lib/python3.5/site-packages/txtorcon/endpoints.py", line 14, in <module>
    from txtorcon.socks import TorSocksEndpoint
  File "/home/christopher/mc/conda-bld/txtorcon_1498584467192/_t_env/lib/python3.5/site-packages/txtorcon/socks.py", line 18, in <module>
    from twisted.protocols import tls
  File "/home/christopher/mc/conda-bld/txtorcon_1498584467192/_t_env/lib/python3.5/site-packages/twisted/protocols/tls.py", line 41, in <module>
    from OpenSSL.SSL import Error, ZeroReturnError, WantReadError
ImportError: No module named 'OpenSSL'
TESTS FAILED: txtorcon-0.19.3-py35_0
```
Maybe I should try to fix the twisted package?